regparser/web/jobs/serializers.py

```diff
@@ -0,0 +1,35 @@
+from regparser.web.jobs.models import ParsingJob
+from rest_framework import serializers
+
+
+class ParsingJobSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = ParsingJob
+        fields = (
+            "cfr_title",
+            "cfr_part",
+            "clear_cache",
+            "destination",  # Unsure about whether this should accept user
+                            # input or be set by the system.
```

Comment:
Definitely should be set by the system in the final product; all sorts of security implications if not.